vinca/templates/activate.sh.in

```diff
@@ -14,5 +14,10 @@ else
     source $CONDA_PREFIX/setup.sh
 fi
 
+case "$OSTYPE" in
```

Comment:
I am unsure if OSTYPE very portable ... haven't seen it used often. 

Maybe we should use `uname` which should give `$(uname) == Darwin` and `$(uname) == Linux`. But idk, maybe OSTYPE is good.

Reply:
It works (at least) in sh, zsh, bash & dash. Should cover most Unix systems?

Reply:
alright, let's keep it. also just tried on Mac with Bash & zsh.